account_analytic_distribution/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Account Analytic Distribution
+=============================
+
+This module is a simpler alternative to official 'account_analytic_plans'
+addon provided by Odoo in version 8.0.
+
+Configuration
+=============
+
+To configure your analytic distributions go to
+Invoicing > Configuration > Analytic Accounting > Analytic Distributions
```

Comment:
Put a little more in this topic, specifying what fields you have to fill, and using `#. ` elements.